ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/README.metadata.json

```diff
@@ -26,9 +33,10 @@
         "Surface"
     ],
     "snippets": [
-        "Surface_Placement.qml",
+        "",
```

Comment:
Ah script doesn't cover name deviations from the folder name(they should be identical in almost all cases or at least I thought so). This is because of the odd situation where we need the QML file first and I was making use of the folder name as the name for the QML file since I didn't come across any other sources where I could guarantee the name. Thanks for catching this.